src/XProfile/VirtoCommerce.XProfile/Schemas/InputIdType.cs

```diff
@@ -0,0 +1,13 @@
+using GraphQL.Types;
+using VirtoCommerce.Platform.Core.Security;
+
+namespace VirtoCommerce.ExperienceApiModule.XProfile.Schemas
```

Comment:
I think this type is redundant because we can pass string parameters as simple arguments

Reply:
will remove